modules/imap/output_modules.php

```diff
@@ -203,10 +203,16 @@ protected function output() {
                             $txt .= '<tr class="header_'.$fld.'"><th>'.$this->trans($name).'</th><td>'.$this->html_safe($value).'</td></tr>';
                         }
                         elseif ($fld == 'reply-to') {
-                            if (str_replace(['<','>'], '', $headers['From']) !== $value) {
-                                $txt .= '<tr class="header_'.$fld.'"><th>'.$this->trans($name).'</th><td>&lt;'.$this->html_safe($value).'&gt;</td></tr>';
-                            } else {
+                            $from = addr_parse($headers['From']);
+                            
+                            $replyEmails = array_map(function ($addr) {
+                                return $addr['email'];
+                            }, process_address_fld($headers['Reply-To']));
+
+                            if (count($replyEmails) === 1 && ($replyEmails[0] === $from['email'])) {
                                 $txt .= '<tr style="display: none;" class="long_header"><th>'.$this->html_safe($name).'</th><td>'.$this->html_safe($value).'</td></tr>';
+                            } else {
+                                $txt .= '<tr class="header_'.$fld.'"><th>'.$this->trans($name).'</th><td>&lt;'.$this->html_safe(join(',', $replyEmails)).'&gt;</td></tr>';
```

Comment:
Having multiple addresses in angle brackets is against RFC: https://www.rfc-editor.org/rfc/rfc5322.html#section-3.4

Reply:
The angle brackets have been removed.
Thanks.